artifacts/centos/centos.go

```diff
@@ -147,11 +147,11 @@ func (c *centos) Tests() []api.ArtifactTest {
 }
 
 // New accepts CentOS 7 and 8 versions. Example patterns are 7-2111, 7-2009, 8.3, 8.4, ...
-func New(release string, additionalLabels map[string]string) *centos {
+func New(release, arch string, additionalLabels map[string]string) *centos {
```

Comment:
What do you think about adding a type for these arch? This way I would be well-known which arch are available/supported. Something like:

```go
type archType string

const X86_64 archType = "x86_64"
const Arm archType = "arm"
```

Reply:
Yep sure. Likely I'll do it like is in your example.

Reply:
I would stick to strings without custom type here. The correct string for ARM64 is likely `aarch64`.